coreos-4/policies/CP-Contingency_Planning/component.yaml

```diff
@@ -482,11 +480,9 @@
         user-level information occures, and how three backup copies are
         maintained.
 
-        Creation of control response/guidance is being tracked in the
-        ComplianceAsCode backlog:
-
-        https://github.com/ComplianceAsCode/redhat/issues/934
+        A complete control response is planned. Progress can be tracked via:
 
+        https://jira.coreos.com/browse/CMP-291
```

Comment:
@redhatrises for a control response that is not applicable, is a complete control response truly planned? Gives the impression the ``not applicable`` may not be accurate

If the language is altered -- eg "To assist with this organizational control, documentation regarding backups is being planned. Progress can be tracked at...."

Reply:
I'll make the language changes... makes more sense for sure.

Reply:
Not sure what is best. Alternatively, since we're enhancing an existing response, perhaps there is no reason to mention the jira? (eg internal sausage making?)

Both seem acceptable.

Reply:
Updated the PR.